test/e2e-tests.sh

```diff
@@ -28,42 +28,101 @@ source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.s
 
 # Helper functions.
 
-# Setup the Knative environment for running tests
+readonly EVENTING_CONFIG="config/"
+
+readonly IN_MEMORY_CHANNEL_PROVISIONER_CONFIG="config/provisioners/in-memory-channel/in-memory-channel.yaml"
+
+readonly GCP_PUBSUB_PROVISIONER_CONFIG="contrib/gcppubsub/config/gcppubsub.yaml"
+readonly GCP_PUBSUB_TEMP_CONFIG="$(mktemp)"
+
+# Constants used for creating ServiceAccount for GCP PubSub provisioner setup if it's not running on Prow.
+readonly PUBSUB_SERVICE_ACCOUNT="eventing_pubsub_test"
+readonly PROJECT_ID="${GCP_PROJECT}"
```

Comment:
This is E2E_PROJECT_ID, so you don't need this extra var. Just set E2E_PROJECT_ID before calling `gcppubsub_setup`.

Reply:
Done.